### PR TITLE
Add pg-admin

### PR DESCRIPTION
**Pull request by pmattmann**
Add container for pg-admin.

Accessable at  [http://localhost:3000/pgadmin](http://localhost:3000/pgadmin)

Username:  admin@ecamp3.ch
Password:  admin

### CODE REVIEW

File: reverse-proxy-nginx.conf [hunk 1]
```diff
@@ -53,6 +53,13 @@ http {
       proxy_set_header Upgrade $http_upgrade;
       proxy_set_header Connection $connection_upgrade;
     }
+
+    location /pgadmin/ {
```
Comment (BacLuc) [inline reverse-proxy-nginx.conf:57]: maybe we now want to expose ecamp only on the localhost interface?
but we can do that in a seprate pr

**Review by BacLuc — APPROVED** [review]

File: .github/workflows/reusable-e2e-tests-run.yml [hunk 2]
```diff
@@ -64,7 +64,7 @@ jobs:
       - run: docker compose -f docker-compose.yml run --rm api migrate-database
 
       # start necessary containers
-      - run: docker compose -f docker-compose.yml up -d api frontend pdf print browserless database docker-host http-cache mail reverse-proxy
+      - run: docker compose -f docker-compose.yml up -d api frontend pdf print browserless database docker-host http-cache mail pg-admin reverse-proxy
```
Comment (carlobeltrame) [inline .github/workflows/reusable-e2e-tests-run.yml:67]: Why do we need pgadmin in CI?

**Review by carlobeltrame — APPROVED** [review]
Maybe document this in the wiki setup docs

File: .github/workflows/reusable-e2e-tests-run.yml (hunk 2 above)
Comment (pmattmann) [reply .github/workflows/reusable-e2e-tests-run.yml:67]: The reverse-proxy container checks whether the configured endpoints (servers) are accessible. If not, this leads to an error.

**Pull request merged**